README.md

```diff
@@ -172,6 +171,9 @@ CLAcroMetrix-D01-X01-X00,DNA,CLAcroMetrix-D01-X01-X00,TCCGGAGA,D702,AGGATAGG,D50
 #### Locally
```

Comment:
Just a thought, you included instructions on how to "run" the cli in lines 107-138 and now you include commands again. I understand that the first part is "general" and here you are actually describing how to use the cli to run the test data. Still, I find it easier if any commands for running it are kept closer together so one does not have to scroll through the README several times.

Reply:
issue #5 created from this comment